frontend/src/components/common/SimpleTable.tsx

```diff
@@ -62,6 +67,56 @@ export interface SimpleTableProps {
   rowsPerPage?: number[];
   emptyMessage?: string;
   errorMessage?: string | null;
+  defaultSortingColumn? : number;
+}
+
+interface ColumnSortButtonProps {
+    defaultSortingColumn?: number;
+    isIncreasingOrder: boolean;
+    sortColIndex: number;
+    setSortColIndex: React.Dispatch<React.SetStateAction<number>>;
+    setIsIncreasingOrder: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+function ColumnSortButtons(props: ColumnSortButtonProps) {
+  const {defaultSortingColumn, isIncreasingOrder,
+         sortColIndex, setSortColIndex, setIsIncreasingOrder} = props;
+  return defaultSortingColumn ?
+    <>
+      {isIncreasingOrder ?
+        <IconButton onClick={() => {
+          setSortColIndex(sortColIndex);
+          setIsIncreasingOrder(false);
+        }
+        }
+        >
+          <Icon icon={menuUp}/>
+        </IconButton> :
+        <IconButton onClick={() => {
+          setSortColIndex(sortColIndex);
+          setIsIncreasingOrder(true);
+        }}
+        >
+          <Icon icon={menuDown}/>
+        </IconButton>
+      }
+    </> :
+    <>
+      <IconButton onClick={() => {
+        setSortColIndex(sortColIndex);
+        setIsIncreasingOrder(false);
+      }}
+      >
+        <Icon icon={menuUp}/>
+      </IconButton>
+      <IconButton onClick={() => {
+        setSortColIndex(sortColIndex);
+        setIsIncreasingOrder(true);
+      }}
+      >
+        <Icon icon={menuDown}/>
+      </IconButton>
+    </>;
```

Comment:
What's the difference between this block and the one above? (Maybe it's still early here and I am not seeing it)

Reply:
in case the table is not default sorted we show both buttons